src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -844,16 +864,10 @@ impl<T: 'static> EventProcessor<T> {
                                 event: Focused(true),
                             });
 
-                            // Issue key press events for all pressed keys
-                            Self::handle_pressed_keys(
-                                &wt,
+                            callback(Event::WindowEvent {
```

Comment:
Almost everything else in this file is reverting #1. Maybe I even should create one PR for revert and another for new changes?